lib/ddp-client.js

```diff
@@ -2,7 +2,17 @@
 
 var util = require("util");
 var events = require("events");
-var WebSocket = require("faye-websocket");
+var WebSocket;
+
+var isBrowserified = (typeof window !== 'undefined');
+if (isBrowserified) {
```

Comment:
I feel like the presence of `window.Websocket` should be the actual test here. And code should be added to detect when it is being run in a non-Websocket implementing browser and do something more helpful that throwing during package load on the absence of `window.Websocket` (which is presumably what happens now in that case)